src/backends/gr.jl

```diff
@@ -217,8 +309,11 @@ end
 
 # ----------------------------------------------------------------
 
-function Base.writemime(io::IO, ::MIME"image/png", plt::PlottingObject{GRPackage})
-  # TODO: write a png to io
+function Base.writemime(io::IO, m::MIME"image/png", plt::PlottingObject{GRPackage})
+  isijulia() || return
```

Comment:
Is it only possible to write a png from IJulia?  I frequently want to do this from the REPL.